modules/proc_manage-clairv4.adoc

```diff
@@ -32,25 +17,21 @@ All supported security databases for Clair v4 are turned on. These databases inc
 * RHEL Oval database
 * SUSE Oval database
 * Ubuntu Oval database
+* Pyup.io (python) database
 
 For information on how Clair does security mapping with the different databases, see
-link:https://quay.github.io/claircore/v0.0.19/severity_mapping.html[ClairCore Severity Mapping].
+link:https://quay.github.io/claircore/concepts/severity_mapping.html[ClairCore Severity Mapping].
 
-[WARNING]
+[NOTE]
 ====
-Because Clair v4 is Technology Preview, don't expect 100% accurate reporting.
-Expect that the presentation of vulnerability results will look different going
-forward than they did with v2. Over time, Clair v4 will produce more results.
+With the release of Red Hat Quay 3.4, the default version of Clair is V4.  This new version V4 is no longer being released as link:https://access.redhat.com/support/offerings/techpreview[Technology Preview] and is supported for production use.  Customers are strongly encouraged to use Clair V4 for with Red Hat Quay 3.4.  It is possible to run both Clair V4 and Clair V2 simultaneously if so desired.  In future versions of Red Hat Quay, Clair V2 will eventually be removed.
 ====
 
-The steps for using Clair v4 alongside an existing {productname} + Clair v2 environment are
-described next.
 
-== Configuring Clair v4
+=== Setting Up Clair on a {productname} OpenShift deployment
+To set up Clair on a new {productname} deployment on OpenShift, it is highly recommended to use the Quay Operator <TODO: ADD LINK TO OPERATOR DOCS>.  By default, the Quay Operator will install a Clair deployment along with your {productname} deployment and configure Clair security scanning automatically.
```

Comment:
I suppose this is correct, but the setup the operator creates is... sub-optimal.

Reply:
ok- let's make it optimal.  Can you get a list of things we need to look at @ see what we can/should do?

Reply:
I think we'll really need to create a Clair operator instead of teaching the Quay operator about Clair's heterogeneous deployments.